Jira plugin

AkitaSp opened the pull request:
Added jira plugin.
Jira plugin provides support for:

Issues:
- Get issue 
- Create issue
- Delete issue
- Assign issue
- Edit issue 

User:
- Get user
- Find users by query
- Assignable users

Worklog:
- Get issue worklogs
- Add worklog
- Delete worklog

Board:
- Get issues for backlog
- Get all boards
- Get issues for board

Auth types:
- Basic Auth (email:api_token)

Rest API version:
v3

Additional info:
- added external lib 'jira.js' to access jira api v3

AkitaSp commented:
Solve for issue #10132

mansukh-tj commented:
@AkitaSp we have reviewed your PR. We have following suggestions-
1. Increase the width of operations dropdown and align it with other inputs 
<img width="793" alt="image" src="https://github.com/user-attachments/assets/b9918c32-6036-4cc1-be70-dfff064e6073">

2. Please share documentation link to obtain the access token
3. Please share a loom video/screen recording showing functionality of all resources and respective operations of the same.

AkitaSp commented:
@mansukh-tj 
- fixed size of fields
- here you can find documentation and video https://drive.google.com/drive/folders/1R6GafILBhF9W2NTP2vkG060KR4L503-5?usp=drive_link

parthy007 reviewed (commented):
<img width="384" alt="Screenshot 2024-08-19 at 12 12 39 PM" src="https://github.com/user-attachments/assets/f29b6aae-54da-4187-9d90-d0efa10a2e09">

Instead of using ```col-md-4``` use ```width``` prop and make it 100%

AkitaSp commented:
@parthy007 , hello! I thinked i already fixed this problem and removed col-md-4 from the class. In the last 2 commits (commit: https://github.com/ToolJet/ToolJet/pull/10223/commits/4db74a0cd32a1c30e585262838705137d0c66cac )
![image](https://github.com/user-attachments/assets/371254e1-407f-48a7-a858-2fca53d90cbb)
